README.md

```diff
@@ -11,33 +16,25 @@
 ERC20Helper is a basic library designed to be used as a middleware between general smart contracts and diverse ERC-20 tokens. It standardizes the behaviours of all commonly used implementations.
```

Comment:
This whole section needs to be updated as it's out of date.

Reply:
My bad, updated the whole file